Jenkinsfile

```diff
@@ -0,0 +1,34 @@
+pipeline {
+   agent { label 'dss-plugin-tests'}
+   environment {
+        HOST = "$dss_target_host"
+    }
+   stages {
+      stage('Run Unit Tests') {
+         steps {
+            sh 'echo "Running unit tests"'
+            sh """
+               make unit-tests
+               """
+            sh 'echo "Done with unit tests"'
+         }
+      }
+//       stage('Run Integration Tests') {
```

Comment:
This needs to be uncommented

Reply:
done in https://github.com/dataiku/dss-plugin-nlp-sentiment-analysis/pull/4/commits/2e39e513360cd79a63ff34fb9ca72856d8a7e52a